apps/game_backend/priv/repo/migrations/20240816155411_add_description_to_game_modes.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GameBackend.Repo.Migrations.AddDescriptionToGameModes do
+  use Ecto.Migration
+
+  def change do
+    alter table(:game_modes) do
+      add :description, :text
```

Comment:
Feel free to squash this into the other migration `AddGameModesTable`

Reply:
Whoops. Nice catch!